apis/storage/volumeclass_types.go

```diff
@@ -21,6 +21,15 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	// Max throughput in bytes per seconds.
+	ResourceBPS corev1.ResourceName = "bps"
```

Comment:
we should calls this `tps` - the value would then end up as
```yaml
tps: 1Gi
```
which is more in line with e.g. Kubernetes' `memory`

Reply:
Sure. What does `tps` stand for?